src/app/questionnaire-item/questionnaire-form-item-text/questionnaire-form-item-text.component.html

```diff
@@ -27,6 +27,7 @@
   mat-raised-button
   color="primary"
   (click)="addItem()"
+  type="button"
```

Comment:
These guys didn't anticipate the arrival of web apps when they decided that the default type is `submit` 😄

Reply:
Yep 😅